rails/locales/ru.yml

```diff
@@ -0,0 +1,125 @@
+ru:
+  activerecord:
+    attributes:
+      doorkeeper/application:
+        name: 'Название'
+        redirect_uri: 'URI для редиректа'
+        scopes: 'Скоупы'
+    errors:
+      models:
+        doorkeeper/application:
+          attributes:
+            redirect_uri:
+              fragment_present: 'Не должен содержать фрагменты'
```

Comment:
Lets start all this validation messages from the downcase char (as it will be concatenated with the attribute name).